src/psyclone/tests/psyir/nodes/omp_directives_test.py

```diff
@@ -549,21 +552,10 @@ def test_omp_loop_directive_validate_global_constraints():
     ''' Test the OMPLoopDirective is inside a OMPParallelRegion and contains
```

Comment:
Please update this docstring as the OMPLoopDirective defines its own parallel region (I assume?). In fact, this test doesn't have anything to do with the collapse setter/getter so please update.

Reply:
Updated the docstring